### ISSUE TITLE
[ICON REQUEST]: Firebase

### ISSUE THREAD

**Issue by fadihere** [devicons/devicon#2204]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

New firebase icon

### Links and sources

New firebase icon should be add:

<img alt="Git" width="100px" src="https://www.gstatic.com/devrel-devsite/prod/vd59826aff4c44c40ae6b66bb6749cdc41cdaad671ce8aa5208d5acf15cc7ec42/firebase/images/lockup.svg" ></img>

https://firebase.google.com/

#### iconname-version-name
<img width="100px" src="previewable_link_to_icon" ></img>
link_to_icon

**Comment by stratusaxarlis** [comment]
firebase exists imho  https://devicon.dev/ https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/firebase/firebase-original-wordmark.svg

**Comment by canaleal** [comment]
> firebase exists imho  https://devicon.dev/ https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/firebase/firebase-original-wordmark.svg
> 

That's correct, Firebase already exists in devicons, but we have to ensure all icons are up to date. 

This new firebase icon will replace the old one essentially.

**Comment by fadihere** [comment]
> > firebase exists imho  https://devicon.dev/ https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/firebase/firebase-original-wordmark.svg
> > 
> 
> That's correct, Firebase already exists in devicons, but we have to ensure all icons are up to date. 
> 
> This new firebase icon will replace the old one essentially.

Exactly, it's not up to date

**Comment by Bharathboy1** [comment]
2026, Still not up to date
